Automattic/wp-calypso
# Add/stepper light importer

#### Changes proposed in this Pull Request

* Moved `import-light` module to the blocks folder
* Registered `importLight` route in stepper framework
* Integrated the static capture screen

The step is under the feature `onboarding/import-light` flag and currently, it's available in the development environment.

#### Testing instructions

- Go to `/setup/importLight?siteSlug={SLUG}` [**from dev env**]
- Check if there is the light importer capture screen

#### Screenshot
<img width="1157" alt="Screenshot 2022-05-23 at 13 44 15" src="https://user-images.githubusercontent.com/1241413/169812238-64866842-ba22-4303-b608-2a5c0a83990f.png">

Related to #63900
